src/test/java/uk/gov/hmcts/cmc/claimstore/constraints/ValidCountyCourtJudgmentValidatorTest.java

```diff
@@ -0,0 +1,108 @@
+package uk.gov.hmcts.cmc.claimstore.constraints;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+import uk.gov.hmcts.cmc.claimstore.models.CountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.ccj.PaymentOption;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleCountyCourtJudgment;
+import uk.gov.hmcts.cmc.claimstore.models.sampledata.SampleRepaymentPlan;
+
+import java.time.LocalDate;
+import javax.validation.ConstraintValidatorContext;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(MockitoJUnitRunner.class)
+public class ValidCountyCourtJudgmentValidatorTest {
```

Comment:
Might be worth having an apiTest that checks this gets invoked?

Reply:
done